.github/workflows/test.yml

```diff
@@ -14,4 +19,25 @@ jobs:
       # check only. not commit.
       - name: Generate README.md
         run: ./generate_readme/main.sh
-      - run: cat README.md
+      - name: Read generated README
+        id: readme
+        run: |
+          content=$(cat README.md)
+          echo "$content"
+          echo "content<<EOF" >> $GITHUB_ENV
+          echo "$content" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+      - name: Comment on PR
+        if: github.event_name == 'pull_request'
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: preview
```

Comment:
If there is an update to a branch, it will update one comment if this `header` does not change.
Therefore, even if the number of push increases, the comment will remain one.